internal/controller/numaresourcesscheduler_controller.go

```diff
@@ -225,6 +238,10 @@ func (r *NUMAResourcesSchedulerReconciler) syncNUMASchedulerResources(ctx contex
 	klog.V(4).Info("SchedulerSync start")
 	defer klog.V(4).Info("SchedulerSync stop")
 
+	if r.PlatformInfo.Platform == platform.OpenShift {
+		preNormalize(&instance.Spec, r.PlatformInfo.Version)
```

Comment:
I would probably have renamed `preNormalize` in something like `platformNormalization` and pushed the `if` block inside the function, but it's largely personal taste. The current approach is not wrong or bad.

Reply:
updated, looks cleaner when the condition is internal in the function. I have updated the condition to apply to hypershift too as I missed that from the start, and updated the tests accordingly. Thanks for raising this point.